numpy/distutils/tests/test_system_info.py

```diff
@@ -0,0 +1,208 @@
+from __future__ import division, absolute_import, print_function
+
+import os
+from tempfile import mkstemp, mkdtemp
+
+import distutils
+from numpy.testing import *
```

Comment:
no need to apologize at all! 
everything I commented on so far is about minor details